packages/model-viewer/src/features/scene-graph/image.ts

```diff
@@ -47,7 +47,7 @@ export class Image extends ThreeDOMElement implements ImageInterface {
       gltfImage: GLTFImage|null) {
     gltfImage = gltfImage ?? {
       name: 'adhoc_image',
-      uri: (texture && texture.image) ? texture.image.uri : 'adhoc_image'
+      uri: (texture && texture.image) ? texture.image.src : 'adhoc_image'
```

Comment:
@timmmeh Another small bug I found (TS should have caught this, but it apparently thinks `image` is `any`.